.travis.yml

```diff
@@ -2,21 +2,21 @@
 language: go
 matrix:
   include:
-    - go: 1.10.x
     - go: 1.11.x
     - go: 1.12.x
+    - go: 1.13.x
 
 before_install:
-  - go get github.com/golang/dep/cmd/dep
+  - export GO111MODULE=on
   - go get golang.org/x/lint/golint
 
 install:
   - # skip
 
 script:
+  - go mod vendor
```

Comment:
Let's not use `go mod vendor`. We don't need to rely on the `vendor` directory when using go modules. I think you should just be able to delete this line.